### ISSUE TITLE
Stop working few minutes after starting : [SPOTIFY] API Request fail on : /v1/me/player

### ISSUE THREAD

**Issue by SuperfrogMe** [skuethe/MMM-Spotify#50]
Hi ! I have an issue with this module.

Here all the logs (I disabled all the other modules for logs)  :
I was playing music during the test for logs

```
[2020-04-26 23:33:59.241] [LOG]    Starting MagicMirror: v2.11.0
[2020-04-26 23:33:59.255] [LOG]    Loading config ...
[2020-04-26 23:33:59.265] [LOG]    Loading module helpers ...
[2020-04-26 23:33:59.269] [LOG]    No helper found for module: alert.
[2020-04-26 23:33:59.364] [LOG]    Initializing new module helper ...
[2020-04-26 23:33:59.366] [LOG]    Module helper loaded: updatenotification
[2020-04-26 23:33:59.367] [LOG]    No helper found for module: clock.
[2020-04-26 23:34:00.005] [LOG]    Initializing new module helper ...
[2020-04-26 23:34:00.006] [LOG]    Module helper loaded: MMM-Spotify
[2020-04-26 23:34:00.007] [LOG]    All module helpers loaded.
[2020-04-26 23:34:00.212] [LOG]    Starting server on port 8080 ...
[2020-04-26 23:34:00.237] [LOG]    Server started ...
[2020-04-26 23:34:00.239] [LOG]    Connecting socket for: updatenotification
[2020-04-26 23:34:00.241] [LOG]    Connecting socket for: MMM-Spotify
[2020-04-26 23:34:00.245] [LOG]    [SPOTIFY_AUTH] Token is fresh.
[2020-04-26 23:34:00.247] [LOG]    [SPOTIFY_AUTH] Token is fresh.
[2020-04-26 23:34:00.248] [LOG]    Sockets connected & modules started ...
[2020-04-26 23:34:00.392] [LOG]    Launching application.
[2020-04-26 23:34:06.863] [LOG]    [MMM-Spotify] Starting
[2020-04-26 23:37:35.817] [LOG]    [SPOTIFY] API Request fail on : /v1/me/player
[2020-04-26 23:37:35.819] [LOG]    Error: getaddrinfo EAI_AGAIN api.spotify.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:60:26) {
  errno: 'EAI_AGAIN',
  code: 'EAI_AGAIN',
  syscall: 'getaddrinfo',
  hostname: 'api.spotify.com'
} undefined
[2020-04-26 23:37:35.843] [LOG]    Whoops! There was an uncaught exception...
[2020-04-26 23:37:35.845] [ERROR]  TypeError: Cannot read property 'statusCode' of undefined
    at Request._callback (/home/pi/MagicMirror/modules/MMM-Spotify/Spotify.js:199:34)
    at self.callback (/home/pi/MagicMirror/node_modules/request/request.js:185:22)
    at Request.emit (events.js:200:13)
    at Request.onRequestError (/home/pi/MagicMirror/node_modules/request/request.js:881:8)
    at ClientRequest.emit (events.js:200:13)
    at TLSSocket.socketErrorListener (_http_client.js:402:9)
    at TLSSocket.emit (events.js:200:13)
    at emitErrorNT (internal/streams/destroy.js:91:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:9)
[2020-04-26 23:37:35.852] [LOG]    MagicMirror will not quit, but it might be a good idea to check why this happened. Maybe no internet connection?
[2020-04-26 23:37:35.853] [LOG]    If you think this really is an issue, please open an issue on GitHub: https://github.com/MichMich/MagicMirror/issues
```

**Comment by eouia** [comment]
Give a more long `updateInterval`, 2000 or 3000.

**Comment by SuperfrogMe** [comment]
> Give a more long `updateInterval`, 2000 or 3000.

Hi eouia !

Below, my config.js for this module :

```
		{
			module: "MMM-Spotify",
			position: "middle_center",
			config: {
				style: "default", // "default" or "mini" available
				control: "hidden", //"default", "hidden" available
				updateInterval: 5*1000, //en millisecondes,
				onStart: null, // disable onStart feature with `null`
				allowDevices: [], //If you want to limit devices to display info, use this.
				// allowDevices: ["RASPOTIFY", "My iPhoneX", "My Home speaker"],
			}
```

**Comment by eouia** [comment]
First, Are you premium(paid) member of Spotify?

**Comment by eouia** [comment]
EAI_AGAIN means the DNS server replied that it cannot currently fulfill the request. 

There is no single solution because it entirely depends on why the DNS server sends that back. Maybe it's overloaded, maybe the network is down, maybe it got the same reply from its upstream server.

In general, the best you can do is wait a while and try again. Hope that helps.

**Comment by SuperfrogMe** [comment]
> First, Are you premium(paid) member of Spotify?

Yes, I have a premium family account.

For the DNS, i have a RPI3 with PiHole.
I tried by deleting all the blocklist, so it wasn't the problem.
Below my DNS configuration in PiHole. 
I will try different configuration to see if the problem recurs.
![image](https://user-images.githubusercontent.com/44573979/80359171-bd832700-887d-11ea-99ca-74643e8899e1.png)

**Comment by vivekpk007** [comment]
i too have the same issue

**Comment by bugsounet** [comment]
corrected with v1.3.0
If any problem, reopen this issue
